src/utils/components/NetworkIcons/LinkStateAbsentIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FC } from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Tooltip, TooltipPosition } from '@patternfly/react-core';
+import { InProgressIcon } from '@patternfly/react-icons';
+
+import './LinkStateIcon.scss';
+
+const LinkStateAbsentIcon: FC = ({}) => {
+  const { t } = useKubevirtTranslation();
+
+  return (
+    <Tooltip
+      content={t('This interface is currently hot-unplugged until the VirtualMachine is migrated.')}
+      position={TooltipPosition.right}
+    >
+      <InProgressIcon className="link-state-icon link-state-icon__absent" />
```

Comment:
we could also consider using "unknown" - not sure what's it's used in Kubvirt in similar cases

Reply:
`absent` is one of the possible values of the `state` field. I would totally agree if it was meant to represent an ambiguous state.